starters/express-apollo-prisma/package.json

```diff
@@ -0,0 +1,89 @@
+{
+	"name": "express-apollo-prisma",
+	"version": "0.0.1",
+	"description": "Express, Apollo and Prisma",
```

Comment:
Regarding my earlier comment, I guess you need to change it here and run the generate kits command.